docs/how_to/layer_manipulator.md

```diff
@@ -8,23 +8,29 @@ Sometimes objects in your data have a systematic shift relative to one another.
 
 ## Instructions
 
-1. Open the [layer_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/layer_manipulator_plugin/) example from the docs. The viewer is initalized in 3D rendering mode. You can 
-   click and drag in the canvas to rotate the view. Note that the two layers are the same image, but not aligned.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/layer_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/layer_manipulator_plugin.py`. 
+
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
+	```bash
+	python layer_manipulator_plugin.py
+	```  
+	
+3. The viewer is initalized in 3D rendering mode. You can click and drag in the canvas to rotate the view. Note that the two layers are the same image, but not aligned.
 	
 	![layers loaded](https://user-images.githubusercontent.com/1120672/207427782-c2b04738-17ed-4963-b6b9-cde6a899ae7f.png)
 	
-2. Select the "image_1" layer in the layer manipulator plugin. We will move layer "image_1" (the green one) to be aligned with layer "image_0".
+4. Select the "image_1" layer in the layer manipulator widget. We will move layer "image_1" (the green one) to be aligned with layer "image_0".
 
 	![select layer](https://user-images.githubusercontent.com/1120672/207428382-3683481f-4acc-4f6a-a108-c497893cf82e.png)
 
-3. Activate the layer manipulator plugin.
+5. Activate the layer manipulator plugin by clicking the "activate" button.
 
 	![activate the plugin](https://user-images.githubusercontent.com/1120672/207428741-3d07f55f-0cc9-4b1b-92eb-6d67d7beac94.png)
 	
-	When the plugin is activated, you will see the manipulator at (0, 0, 0) on image_1.
+	When the plugin is activated, you will see the manipulator at (0, 0, 0) on image_1. Note that depending on your versions of `napari-threedee` and `napari` you may see subtle visual differences between your viewer and the screenshots/videos here.
```

Comment:
```suggestion
	When the plugin is activated, you will see the manipulator at (0, 0, 0) on image_1 (the green image). Note that depending on your versions of `napari-threedee` and `napari` you may see subtle visual differences between your viewer and the screenshots/videos here.
```

Suggest: adding "the green image", just for clarity (I know you say this in other steps, but worthwhile repeating it)

Also, do we really need the "subtle visual differences" warning? I feel like you're trying to ward off a criticism that hasn't actually happened (and might not actually happen)